src/open_inwoner/components/templates/components/List/ListItem.html

```diff
@@ -1,7 +1,12 @@
 {% load link_tags %}
 <li class="list-item{% if active %} list-item--active{% endif %}">
     {% if href %}<a class="link" href="{{ href }}">{% endif %}
-    <h4 class="h4">{{ text }}</h4>
+    {% if caption %}<p class="p p--muted">{{ caption }}</p>{% endif %}
```

Comment:
I don't think it should be muted color:
* `  --color-mute: var(--color-gray);` and our `-color-gray` is very light and not very readable
* according to the design it should be the same color as headers in the action list, for example, which we implemented much darker

Reply:
I agree that the contrast is too low, but aren't the colors configurable and therefore; shouldn't the settings be changed in such a way that muted texts remain readable, even though they require less attention that the more relevant information?

Changing the implementation to suit the configuration would IMO make the configuration unreliable.

Reply:
Do you mean that we can configure colors in the admin? I think we can't configure mute color there

Anyway I think header should be styled the same way as headers in action list, it will solve all problems